ot3Image.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+branch=  
+while getopts ":b:" opt; do
+  case $opt in
+    b)
+      branch=$OPTARG >&2
+      echo $OPTARG
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Option -$OPTARG requires an argument." >&2
+      exit 1
+      ;;
+  esac
+done
+shift $((OPTIND - 1))
+
+
+mkdir -p docker
+cd docker
+mkdir -p oe-core/build
+
+echo $branch
+curl "https://raw.githubusercontent.com/Opentrons/meta-opentrons/${branch}/Dockerfile" > Dockerfile
```

Comment:
Do we need this machinery? Couldn't we just build the branch that's currently checked out?